pkg/runtime/runtime-manifest.go

```diff
@@ -88,8 +88,20 @@ func (m *RuntimeManifest) loadBundle() error {
 	return nil
 }
 
+func (m *RuntimeManifest) GetInstallationNamespace() string {
+	before, _, found := strings.Cut(m.Getenv(config.EnvInstallationName), "/")
```

Comment:
I think it would be helpful to make it easy for people to access the name and namespace without having to use the template language or parse the CNAB_INSTALLATION environment variable.

How about we introduce `PORTER_INSTALLATION_NAME` and `PORTER_INSTALLATION_NAMESPACE` that have the values we want, so that at runtime we don't need to parse the CNAB_INSTALLATION variable?

The place to populate those environment variables is in this function below. You can make a new function similar to `r.AddFiles` which can then set `op.Env` using `args.Installation`:

https://github.com/getporter/porter/blob/fe0146b9c8a330eeb63a265576b1e7f7eeba6a4e/pkg/cnab/provider/action.go#L51-L57

Reply:
Yes, this makes sense. Then we don't have to change the existing definition of installation.name. And as you mention, the values are available directly to action scripts without having to pass them in through template vars. I'll go ahead and update this PR, although its name is no longer accurate. Should I close this and create a new PR?

Reply:
I've updated the PR with those suggestions. As it stands, the relevant env in the runtime is:
```
CNAB_INSTALLATION_NAME=inst_namespace/inst_name
PORTER_INSTALLATION_NAMESPACE=inst_namespace
PORTER_INSTALLATION_NAME=inst_name
```
And template variables installation.namespace and installation.name are derived from PORTER_INSTALLATION_NAMESPACE and PORTER_INSTALLATION_NAME.